src/index.js

```diff
@@ -37,22 +37,20 @@ function installFiles(sourceDir, done) {
   switch (npmv.majorVersion()) {
     case '1':
       console.log("[install-files]: WARNING: NPMv1 is not officially supported; unexpected results could occur. Consider upgrading to v2 or later");
+      /* falls through */
     case '2':
       source = sourceDir;
       target = fileInstallingPackagePath && hostPackageDir(fileInstallingPackagePath);
       break;
     case null:
-      console.log("[install-files]: WARNING: Could not determine NPM version"); //Fall back to default
+      console.log("[install-files]: WARNING: Could not determine NPM version");
+      /* falls through */
     default:
       source = path.join(fileInstallingPackagePath, 'node_modules', process.env.npm_package_name, sourceDir);
-      target = fileInstallingPackagePath
+      target = fileInstallingPackagePath;
   }
 
-  if (fileInstallingPackagePath.match(".+" + process.env.npm_package_name + "$")) {
```

Comment:
@GoGoCarl in what circumstances did you find this to be necessary when you added this in https://github.com/mixmaxhq/install-files/pull/5? As far as I can tell it's always `true` in both npm 2 and 3 as I describe [here](https://github.com/mixmaxhq/install-files/pull/6/commits/60a96146f85ccd55876a79487f983f9529aace93).